src/test/java/org/testobject/appium/junit/RemoteWebDriverCalculatorSuiteTestJUnit.java

```diff
@@ -33,7 +33,7 @@ public void setup() throws MalformedURLException {
         capabilities.setCapability(TestObjectCapabilities.TESTOBJECT_API_KEY, watcher.getApiKey());
         capabilities.setCapability(TestObjectCapabilities.TESTOBJECT_TEST_REPORT_ID, watcher.getTestReportId());
 
-        driver = new RemoteWebDriver(TestObjectCapabilities.TESTOBJECT_APPIUM_ENDPOINT, capabilities);
+        driver = new RemoteWebDriver(watcher.getTestObjectOrLocalAppiumEndpointURL(), capabilities);
```

Comment:
Can't we just turn this into getAppiumEndpointURL and put java doc? It is a really ugly looking method name. Maybe you should ask this to Andreas probably